tests/SpecTests/Operation.php

```diff
@@ -568,8 +568,7 @@ private function executeForTestRunner(FunctionalTestCase $test, Context $context
                  * instead exposes a bool to let us know whether a transaction
                  * is currently in progress. This code may fail down the line
                  * and should be adjusted once PHPC-1438 is implemented. */
-                $expected = in_array($this->arguments['state'], ['in_progress', 'starting']);
-                $test->assertSame($expected, $args['session']->isInTransaction());
+                $test->assertSame($this->arguments['state'], $args['session']->getTransactionState());
```

Comment:
I just realized this isn't documented. Added a label on [PHPC-1438](https://jira.mongodb.org/browse/PHPC-1438).